app/black_box/black_box.rb

```diff
@@ -38,6 +35,8 @@ def comment_quality_score(comment)
     def calculate_spaminess(story, function_caller = FunctionCaller)
       # accepts comment or article as story
       return 100 unless story.user
+      return 0 if ENV["AWS_SDK_KEY"].blank? # Skip this if we don't have a private spam score for now
+      return 0 if ENV["AWS_SDK_KEY"] == "foobarbaz" # Also skip if placeholder
```

Comment:
This is just because at the moment I think we _need_ to have this set in prod, and are using `foobarbaz` at the moment in these cases.